src/Groups/libraries/smallgroups.jl

```diff
@@ -70,7 +70,15 @@ function all_small_groups(n::Int, L...)
 
    K = GAP.Globals.CallFuncList(GAP.Globals.AllSmallGroups,L1)
    return [_get_type(K[i])(K[i]) for i in 1:length(K)]          # GAP.julia_to_gap(K) does not work
-end
+#T what does this comment mean?
+
+#T problem:
 
+#T all_small_groups( 60, issimple ) -> Array{PermGroup,1}
+#T all_small_groups( 60 )  -> Array{Oscar.GAPGroup,1}
+#T all_small_groups( 59 )  -> Array{PcGroup,1}
 
+#T Do we want this???
```

Comment:
I will partially deal with this in my PR for the group libraries, by turning this into an iterator.

Reply:
Thanks.